src/sentry/models/files/utils.py

```diff
@@ -47,6 +47,7 @@ def get_size_and_checksum(fileobj, logger=nooplogger):
             break
         size += len(chunk)
         checksum.update(chunk)
+    fileobj.seek(0)
```

Comment:
I hope this does not break something else that implicitly relies on `get_size_and_checksum` consuming the file.